### PR TITLE
pid1: introduce dbus properties WatchdogDevice and friends

### LINKED ISSUE #24665: Hardware Watchdog information
### Component

systemd

### Is your feature request related to a problem? Please describe

When setting up a hardware watchdog, it would be helpful if there was some way I could ask systemd if the watchdog was configured and any applicable settings.

Because systemd has the `/dev` node open, `wdctl` can't get information about it.

### Describe the solution you'd like

Output similar to what `wdctl` provides from one of the existing tools would be great.  Exposure over dbus might also be nice.

### Describe alternatives you've considered

_No response_

### The systemd version you checked that didn't have the feature you are asking for

_No response_

### PR DESCRIPTION
**Pull request by yuwata**
Closes #24665.

### CODE REVIEW

File: src/core/dbus-manager.c [hunk 1]
```diff
@@ -2783,6 +2788,11 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_PROPERTY("DefaultOOMPolicy", "s", bus_property_get_oom_policy, offsetof(Manager, default_oom_policy), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("DefaultOOMScoreAdjust", "i", property_get_oom_score_adjust, 0, SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("CtrlAltDelBurstAction", "s", bus_property_get_emergency_action, offsetof(Manager, cad_burst_action), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("WatchdogDevice", "s", property_get_watchdog_device, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("WatchdogTimeoutUsec", "t", property_get_watchdog_timeout, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("WatchdogPreTimeoutUsec", "t", property_get_watchdog_pretimeout, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("WatchdogLastPingTimestamp", "t", property_get_watchdog_last_ping_realtime, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```
Comment (bluca) [inline src/core/dbus-manager.c:2794]: `BUS_PROPERTY_DUAL_TIMESTAMP`?

**Review by bluca — APPROVED** [review]

File: src/core/dbus-manager.c (hunk 1 above)
Comment (yuwata) [reply src/core/dbus-manager.c:2794]: Currently, we do not save the timestamp in Manager. Hence, we cannot use the macro.

Comment (yuwata) [reply src/core/dbus-manager.c:2794]: @bluca Is it ok to upgrade the green label? Or do I change that the time stamp is stored to Manager?

Comment (bluca) [reply src/core/dbus-manager.c:2794]: No that's fine

**Pull request merged**